Userland/Libraries/LibWeb/Forward.h

```diff
@@ -70,10 +70,13 @@ class BorderRadiusStyleValue;
 class BorderStyleValue;
 class CalculatedStyleValue;
 class Clip;
+class CSSConditionRule;
+class CSSKeyframeRule;
+class CSSKeyframesRule;
 class ColorStyleValue;
+class CompositeStyleValue;
 class ConicGradientStyleValue;
 class ContentStyleValue;
-class CSSConditionRule;
 class CSSFontFaceRule;
```

Comment:
Nit: This sorting confuses me.

Reply:
Yeah I suppose I should move all the CSS* to the top, my bad.